san/tests/test_sanbase_integration.py

```diff
@@ -73,6 +73,25 @@ def test_erc20_projects():
     assert len(result) > 0
     assert len(result[result.slug == "bitcoin"]) == 0
 
+@attr('integration')
+def test_ordinary_function():
```

Comment:
A single test case for testing more than one query is not a good idea, also having a loop + conditional in a test. 

You can declare some constants
```python
METRICS_USING_ETHEREUM = ["gas_used", "miners_balance", "mining_pools_distribution"]
METRICS_USING_SANTIMENT = ["historical_balance", "top_holders_percent_of_total_supply", "social_dominance"]
```
and use [nose test generators](https://nose.readthedocs.io/en/latest/writing_tests.html#test-generators)

That way we will have test case for every function and simple iteration w/o conditional.

Reply:
You're right about the single test case - all queries. I've made new tests, which explicitly test out one query at a time. (did not use nose test generators though)